Altis_Life.Altis/core/shops/fn_chopShopSell.sqf

```diff
@@ -7,14 +7,12 @@
     Sells the selected vehicle off.
 */
 disableSerialization;
-private ["_control","_price","_vehicle","_nearVehicles","_price2","_chopable"];
+private ["_control","_price","_vehicle","_price2"];
 _control = CONTROL(39400,39402);
 _price = _control lbValue (lbCurSel _control);
 _vehicle = _control lbData (lbCurSel _control);
 _vehicle = call compile format ["%1", _vehicle];
-_chopable = LIFE_SETTINGS(getArray,"chopShop_vehicles");
```

Comment:
Sorry didn't spot this at first glance, Should probably keep the chopable whitelist and add a check in this file since it is no longer being filtered via nearestObjects

Reply:
But we already got chopable whitelist on fn_chopShopMenu.sqf. So non-whitelisted chopable vehicles won't be listed on the Chop_Shop dialog. Isn't that right ?

Reply:
Ah yes my bad that is a good point. I guess that makes sense as it seems the intention were that the indexes would be the same in both these scripts which seems to not be the case in all circumstances